schedule/yast/textmode/ha_textmode_skip_registration_bsc1208266.yaml

```diff
@@ -0,0 +1,28 @@
+name: ha_textmode_skip_registration
```

Comment:
For case ' ha_textmode_skip_registration ', it doesn't have default yaml file at x86_64, I need create a new one for aarch64. As I have ticket 'https://progress.opensuse.org/issues/124035' to do this task. I'd like to create default yaml file at other PR instead of this one.